snapshots/src/lp-helpers.ts

```diff
@@ -4,150 +4,161 @@ import { BigNumber } from "@ethersproject/bignumber";
 
 // TYPES
 export interface IStartAndEndBlock {
-  epochStartBlock: number | null;
-  epochEndBlock: number | null;
-  marketStartBlock: number | null;
-  marketEndBlock: number | null;
-  rewardMarketEndBlock: number | null;
+    epochStartBlock: number | null;
+    epochEndBlock: number | null;
+    marketStartBlock: number | null;
+    marketEndBlock: number | null;
+    rewardMarketEndBlock: number | null;
+    rewardMarketStartBlock: number | null;
 }
 
 export interface LpMarketInfo {
-  marketMaker: string;
-  howToCalculate: LpCalculation;
-  amount: number;
-  rewardMarketEndDate: number | null;
+    marketMaker: string;
+    howToCalculate: LpCalculation;
+    amount: number;
+    rewardMarketEndDate: number | null;
+    rewardMarketStartDate: number | null;
+    eventStartDate: number | null;
+    preEventPercent: number | null;
 }
 
 export enum LpCalculation {
-  PerBlock = "perBlock",
-  PerMarket = "perMarket",
+    PerBlock = "perBlock",
```

Comment:
Should we remove this now?

Reply:
I don't think we need to. I could see it coming back and that logic is frequently touched. Not worth it right now IMO.